js/main.js

```diff
@@ -4,14 +4,53 @@ var $imageURL = document.querySelector('input[name="imageUpload"]');
 var $textArea = document.querySelector('textarea[name="notes"]');
 var $title = document.querySelector('input[name="title"]');
 var $img = document.querySelector('img');
+var $entriesDisplay = document.querySelector('.entries-list-container');
+var $entriesNav = document.querySelector('h4.nav-item');
+var $new = document.querySelector('.new-anchor');
+var $entryForm = document.querySelector('div.entry-form');
+
+if (classNames.whoIsHidden === 'entryForm') {
+  // set the classes properly
+  $entriesDisplay.className = 'entries-list-container';
+  $entryForm.className = 'hidden';
+}
+
+if (classNames.whoIsHidden === 'entriesDisplay') {
+  // set the classes properly
+  $entriesDisplay.className = 'hidden';
+  $entryForm.className = 'entry-form';
+}
+
+$entriesNav.addEventListener('click', function (event) {
+  $entriesDisplay.className = 'entries-list-container';
+  $entryForm.className = 'hidden';
+
+  // update Local Storage
+  classNames.whoIsHidden = 'entryForm';
+  var classNamesJSON = JSON.stringify(classNames);
+  localStorage.setItem('classNames', classNamesJSON);
+});
+
+$new.addEventListener('click', function (event) {
+  $entriesDisplay.className = 'hidden';
+  $entryForm.className = 'entry-form';
+
+  // update Local Storage
+  classNames.whoIsHidden = 'entriesDisplay';
+  var classNamesJSON = JSON.stringify(classNames);
+  localStorage.setItem('classNames', classNamesJSON);
```

Comment:
Here, you can just update `data.view` to the correct view name so that it is automatically stored in local storage.